server/src/main/java/org/apache/uniffle/server/LocalStorageChecker.java

```diff
@@ -100,38 +100,44 @@ public boolean checkIsHealthy() {
     Map<StorageInfo, CompletableFuture<Void>> futureMap = new HashMap<>();
     for (StorageInfo storageInfo : storageInfos) {
       CompletableFuture<Void> storageCheckFuture =
-          CompletableFuture.supplyAsync(
-              () -> {
-                if (!storageInfo.checkStorageReadAndWrite()) {
-                  storageInfo.markCorrupted();
-                  corruptedDirs.incrementAndGet();
-                  return null;
-                }
-
-                long total = getTotalSpace(storageInfo.storageDir);
-                long availableBytes = getFreeSpace(storageInfo.storageDir);
-
-                totalSpace.addAndGet(total);
-                wholeDiskUsedSpace.addAndGet(total - availableBytes);
-                long usedBytes = getServiceUsedSpace(storageInfo.storageDir);
-                serviceUsedSpace.addAndGet(usedBytes);
-                storageInfo.updateServiceUsedBytes(usedBytes);
-                storageInfo.updateStorageFreeSpace(availableBytes);
-
-                boolean isWritable = storageInfo.canWrite();
-                ShuffleServerMetrics.gaugeLocalStorageIsWritable
-                    .labels(storageInfo.storage.getBasePath())
-                    .set(isWritable ? 0 : 1);
-                ShuffleServerMetrics.gaugeLocalStorageIsTimeout
-                    .labels(storageInfo.storage.getBasePath())
-                    .set(0);
-
-                if (storageInfo.checkIsSpaceEnough(total, availableBytes)) {
-                  num.incrementAndGet();
-                }
-                return null;
-              },
-              workers);
+          CompletableFuture.runAsync(
```

Comment:
This should not be covered, because it is wrapped by the `CompletableFutureExtension`, and then exception will be thrown

Reply:
Done.